tools/labelme/labelme.go

```diff
@@ -0,0 +1,50 @@
+package labelme
+
+import (
+	"encoding/json"
+
+	"github.com/linkernetworks/annotation"
+)
+
+type LabelME struct {
+	Shapes    []Shape `bson:"shapes" json:"shapes"`
+	LineColor [4]int  `bson:"lineColor" json:"lineColor"`
+	FillColor [4]int  `bson:"fillColor" json:"fillColor"`
+	ImagePath string  `bson:"imagePath" json:"imagePath"`
+	ImageData string  `bson:"imageData" json:"imageData"`
+}
+
+type Shape struct {
+	Label     string  `bson:"label" json:"label"`
+	LineColor *[4]int `bson:"line_color" json:"line_color"`
+	FillColor *[4]int `bson:"fill_color" json:"fill_color"`
+	Points    []Point `bson:"points" json:"points"`
+}
+
+type Point [2]int
+
+func NewLabelME(imgPath string, label string, lineColor [4]int, fillColor [4]int, imgData []byte) *LabelME {
+	return &LabelME{
+		ImagePath: imgPath,
+		LineColor: lineColor,
+		FillColor: fillColor,
+		ImageData: string(imgData),
+	}
+}
+
+func (l *LabelME) AddAnnotation(ann annotation.PolygonAnnotation, lineColor *[4]int, fillColor *[4]int) {
```

Comment:
we should avoid importing the "annotation" package. Please rename AddAnnotation to "AddShape" and only accept "Shape" type as the argument.